### PR TITLE
Story 4.1 style update

### PR DESCRIPTION
**Pull request by ChrisTregaskis**
Changes implemented following PO comments as well as general code clean
I've added react-spring so please npm install to grab new dependency

- red text for error response for public facing and black response message text for admin areas
- tick (animated) for confirmation of successful sign in
- tick (animated) for confirmation of successful sign out
- tick (animated) for confirmation of successful sign out from visitorSignOutModal
- fix visitorSignOutModal bug where the modal reappeared after you closed it
- currently no one signed in message, less technical (no visitors currently signed in)
- amend height for all response messages so that elements in DOM don't shift
- amended setTimeOut time for messages to 5 seconds to allow user more time to read message
- subtle and simple transition animation for modals
- subtle fade in for admin page table so user doesn't see the table load after the container it self
- grey out the background for modals
- make buttons consistent styling
- refactor globally set tags in css ('div', 'form', 'span' etc)
- removed 'text-align: centre' from global body tag and updated elements where necessary 
- changed colour of admin page background to subtly show user is logged into an authenticated area
- moved logout button to inside the container
- fixed bug where clicking admin button whilst visitorSignOutModal active makes keypad appear underneath
- upon code review, amended all hex codes to be consistent

### CODE REVIEW

File: app/src/Components/AdminPage/AdminContainer/adminContainer.css [hunk 1]
```diff
@@ -1,10 +1,10 @@
 .adminContainer {
     width: 700px;
+    text-align: center;
     min-height: 500px;
     background-color: #ffffff;
```
Comment (robings) [inline app/src/Components/AdminPage/AdminContainer/adminContainer.css:5]: Capitals on hex color defs for consistency.

File: app/src/Components/AdminPage/LogOutBtn/logOutBtn.css [hunk 2]
```diff
@@ -1,12 +1,12 @@
 .logOutBtn {
-    position: absolute;
-    top: 3%;
-    right: 5%;
+    float: right;
+    margin-right: 40px;
     background-color: #F27424;
     color: #ffffff;
```
Comment (robings) [inline app/src/Components/AdminPage/LogOutBtn/logOutBtn.css:5]: Capitals on hex color defs for consistency.

File: app/src/Components/AdminPage/adminPage.css [hunk 3]
```diff
@@ -1,3 +1,5 @@
 h1.adminPage {
-    padding: 20px;
-}
\ No newline at end of file
+    padding: 30px;
+    color: #ffffff;
```
Comment (robings) [inline app/src/Components/AdminPage/adminPage.css:3]: Capitals on hex color defs for consistency.

File: app/src/Components/AdminPage/VisitorsTable/visitorsTable.css [hunk 4]
```diff
@@ -1,5 +1,5 @@
 .visitorsTable {
-    padding: 40px;
+    padding: 10px 30px 30px 30px;
     color: #222;
```
Comment (robings) [inline app/src/Components/AdminPage/VisitorsTable/visitorsTable.css:3]: For consistency, hex color definition should be #222222

File: app/src/Components/LandingPage/AdminBtn/AdminBtn.css [hunk 5]
```diff
@@ -5,16 +5,9 @@
     background-color: #F27424;
     color: #ffffff;
```
Comment (robings) [inline app/src/Components/LandingPage/AdminBtn/AdminBtn.css:6]: Capitals on hex color defs for consistency.

File: app/src/Components/LandingPage/AdminModal/AdminModal.css [hunk 6]
```diff
@@ -1,5 +1,7 @@
 .adminModal {
-    font: 700 35px Arial, sans-serif;
+    font-weight: 700;
+    font-size: 35px;
+    text-align: center;
     width: 700px;
     min-height: 500px;
     background-color: #ffffff;
```
Comment (robings) [inline app/src/Components/LandingPage/AdminModal/AdminModal.css:7]: Capitals on hex color defs for consistency.

File: app/src/Components/LandingPage/AdminModal/AdminModal.css [hunk 7]
```diff
@@ -50,22 +45,28 @@
 
 .keypadBtn {
     background-color: #288FB0;
-    border: 2px solid #295074;
+    border: 1px solid #295074;
     border-radius: 5px;
     color: #ffffff;
```
Comment (robings) [inline app/src/Components/LandingPage/AdminModal/AdminModal.css:50]: Capitals on hex color defs for consistency.

File: app/src/Components/LandingPage/AdminModal/AdminModal.css [hunk 8]
```diff
@@ -50,22 +45,28 @@
...
+    justify-content: center;
+    align-items: center;
 }
 
 .logInBtn {
     background-color: #94BA66;
-    border: 2px solid #295074;
+    border: 1px solid #295074;
     border-radius: 5px;
     color: #ffffff;
```
Comment (robings) [inline app/src/Components/LandingPage/AdminModal/AdminModal.css:63]: Capitals on hex color defs for consistency.

File: app/src/Components/LandingPage/AdminModal/AdminModal.css [hunk 9]
```diff
@@ -75,12 +76,15 @@
     width: 40px;
     height: 40px;
     background-color: #F27424;
-    border: 2px solid #295074;
+    border: 1px solid #295074;
     border-radius: 5px;
     color: #ffffff;
```
Comment (robings) [inline app/src/Components/LandingPage/AdminModal/AdminModal.css:81]: Capitals on hex color defs for consistency.

File: app/src/Components/LandingPage/AdminModal/AdminModal.css [hunk 10]
```diff
@@ -93,15 +97,17 @@
...
-    line-height: 65px;
-}
-
 .responseMessage {
-    font-size: 1rem;
+    font-size: 1.3rem;
+    font-weight: 500;
     padding: 0;
     margin: 0;
     color: #e74c3c;
```
Comment (robings) [inline app/src/Components/LandingPage/AdminModal/AdminModal.css:105]: Capitals on hex color defs for consistency.

File: app/src/Components/LandingPage/LandingPage.css [hunk 11]
```diff
@@ -1,13 +1,17 @@
-body {
-    background-color: #288FB0;
+.landingPage h1 {
+    color: #ffffff;
```
Comment (robings) [inline app/src/Components/LandingPage/LandingPage.css:2]: Capitals on hex color defs for consistency.

File: app/src/Components/LandingPage/MainContainer/MainContainer.css [hunk 12]
```diff
@@ -15,6 +16,21 @@
     0 100px 80px rgba(0, 0, 0, 0.12);
 }
 
+.responseMessageFailed {
+    color: #e74c3c;
```
Comment (robings) [inline app/src/Components/LandingPage/MainContainer/MainContainer.css:20]: Capitals on hex color defs for consistency.

File: app/src/Components/AdminPage/LogOutBtn/logOutBtn.css (hunk 2 above)
Comment (ChrisTregaskis) [reply app/src/Components/AdminPage/LogOutBtn/logOutBtn.css:5]: Thanks for this, done for all hex codes :)

File: app/src/Components/LandingPage/MainContainer/MainContainer.css [hunk 13]
```diff
@@ -15,6 +16,21 @@
...
+}
+
+.successTick {
+    height: 400px;
+    width: 400px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: lawngreen;
```
[the remark this answers is not in the document]
Comment (ChrisTregaskis) [reply app/src/Components/LandingPage/MainContainer/MainContainer.css:30]: Thanks for sharing the hexcode! really convenient

File: app/src/Components/LandingPage/VisitorSignOutModal/VisitorSignOutModal.js [hunk 14]
```diff
@@ -70,24 +82,40 @@ class VisitorSignOutModal extends React.Component
...
+                this.setState({success: false});
+
+                if (this.props.signOutModalVisible) {
+                    this.props.updateSignOutModalVisible();
+                }
+
             }
-        }, 3000);
-        this.setState({response : newResponse})
+        }, 2500);
```
[the remark this answers is not in the document]
Comment (ChrisTregaskis) [reply app/src/Components/LandingPage/VisitorSignOutModal/VisitorSignOutModal.js:117]: honestly no. The tick total animation is 2 seconds so as to make the user experience smoother. In this instance, as the tick lasts 2 seconds, I made the signout box last 500 ms longer so they fade out at different times... that said, maybe they should just fade out at the same time! I'll change it to match the tick at 2000ms.

File: app/src/Components/LandingPage/VisitorSignOutModal/VisitorSignOutModal.js [hunk 15]
```diff
@@ -1,5 +1,6 @@
 import React from "react";
```
[the remark this answers is not in the document]
Comment (ChrisTregaskis) [reply app/src/Components/LandingPage/VisitorSignOutModal/VisitorSignOutModal.js:1]: Interesting! I haven't been able to replicated it. 
I'll do the merge, and at some point if you have time to demonstrate it to me, that'd be great! 
Thanks

File: app/src/Components/LandingPage/VisitorSignOutModal/VisitorSignOutModal.js [hunk 16]
```diff
@@ -1,5 +1,6 @@
...
 import './VisitorSignOutModal.css'
```
[the remark this answers is not in the document]
Comment (ChrisTregaskis) [reply app/src/Components/LandingPage/VisitorSignOutModal/VisitorSignOutModal.js:2]: agreed. Nice find!! All sorted as above. If admin button clicked, visitorSignOutModal disappears and keypad login appears.

**Pull request merged**